src/ClusterBootstrap/ctl.py

```diff
@@ -395,10 +436,14 @@ def run_command(args, command):
     parser.add_argument("--nocache",
                         help="Build docker without cache",
                         action="store_true")
+    parser.add_argument("--admin",
+                        help="Name of admin that execute this script")
     parser.add_argument("command",
                         help="See above for the list of valid command")
     parser.add_argument('nargs', nargs=argparse.REMAINDER,
                         help="Additional command argument")
+    parser.add_argument(
+        '-d', '--dryrun', help='Dry run -- no actual execution', action="store_true")
```

Comment:
This parameter doesn't seem used?

Reply:
it's used. previously we don't have this parameter in ctl.py because we basically only set need_output in other files, not ctl.py. Now we need output to filter annotated nodes, so this parameter is used.